utils/base16384/Makefile

```diff
@@ -0,0 +1,45 @@
+# SPDX-Identifier-License: GPL-3.0-or-later
+#
+# Copyright (C) 2022 Fumiama Minamoto <fumiama@foxmail.com>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=base16384
+PKG_VERSION:=2.2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/fumiama/base16384/tar.gz/v$(PKG_VERSION)
+PKG_HASH:=c9acc69b9f64cfbd9a7062ab8cac0c4814a746c2124e5f00408b13f368f47764
+
+PKG_MAINTAINER:=Fumiama Minamoto <fumiama@foxmail.com>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+
+CMAKE_INSTALL:=1
```

Comment:
looks like your project contains no header files, so this can be removed.

Reply:
Then if I added the `Package/libbase16384`, would this line be preserved?

Reply:
oh yes

Reply:
Thanks, I will push a fix later.